vtex-apps/shipping-calculator
# CHK-544: Use ShippingForm component from checkout-shipping

#### What problem is this solving?

Updates to use the same component as Checkout does, to reduce maintenance burden and complexity.

#### How to test it?

[Workspace](https://addr--checkoutio.myvtex.com/cart/add?sku=289).

You can see in the cart page that the component is the same as the one used in Checkout.

#### Screenshots or example usage:

#### Describe alternatives you've considered, if any.

N/A

#### Related to / Depends on

N/A
